Common/Orders/Fills/InterfaceFillModel.cs

```diff
@@ -1,33 +1,32 @@
 ﻿/*
  * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
  * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
- * 
- * Licensed under the Apache License, Version 2.0 (the "License"); 
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
- * 
+ *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
 */
 
-using QuantConnect.Orders;
-using QuantConnect.Orders.Fees;
-using QuantConnect.Orders.Fills;
-using QuantConnect.Orders.Slippage;
-using QuantConnect.Securities.Equity;
-using QuantConnect.Securities.Forex;
 
-namespace QuantConnect.Securities.Interfaces
+namespace QuantConnect.Orders.Fills
 {
     /// <summary>
-    /// Security transaction model interface for QuantConnect security objects
+    /// Represents an interface that simulates order fill events
     /// </summary>
-    /// <seealso cref="EquityTransactionModel"/>
-    /// <seealso cref="ForexTransactionModel"/>
-    public interface ISecurityTransactionModel : IFillModel, IFeeModel, ISlippageModel
+    /// <remarks>Users should override base class <see cref="FillModel"/></remarks>
+    public interface InterfaceFillModel
```

Comment:
Just a minor comment: maybe this interface could be named `IFillModelInterface` or `IBaseFillModel` for naming convention consistency?
https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/8bc1fexb(v=vs.71)

Reply:
Whoops! Good idea about the naming convention, I'll rename to `IFillModelInterface` 😅